pr-2.html

```diff
@@ -5,108 +5,117 @@
-        </p>
-      </td>
-    </tr>
-    <!-- form & image section -->
-    <tr>
-      <!-- form -->
-      <td>
-        <p> Fill in the form to create your account.</p>
-        <form id="registration-form" action="#registration-form" method="get">
-          &nbsp;&nbsp;
-          <label for="name">User Name:</label>
-          <input type="text" id="username" name="username" pattern="[a-z0-9_]{1,10}"
-            title="User name must be 1-10 characters long and can only contain lowercase letters, numbers, and underscores"
-            required placeholder="Enter your username">
-          <br><br>
+  <div class="container">
+    <table>
+      <tr>
+        <td colspan="2">
+          <h1>- Registration Page -</h1>
+          <h3>Register Your Free Account <a href="#registration-form">Here.</a></h2>
+        </td>
+      </tr>
+      <!-- video section -->
+      <tr>
+        <!-- video -->
+        <td colspan>
+          <h2>Quotes of the Day</h2>
+          <p class="quotes">"And the world keeps spinning around."</p>
+          <video controls width="400" autoplay loop>
+            <source src="./media/earth-video.mp4" type="video/mp4">
+            Your browser does not support the video element.
+          </video>
+        </td>
+        <!-- iframe -->
+        <td colspan>
+          <h2>Today's Highlight</h2>
+          <p>Enjoy this highlight of the legendary Sage Main.</p>
+          <iframe src="https://www.youtube.com/embed/MKBS3_i_SO0?autoplay=1&mute=1" width="420" height="315"
+            allowfullscreen></iframe>
+        </td>
+      </tr>
+      <!-- audio section -->
+      <tr>
+        <td colspan="2">
+          <p>
+            &#127925;&#127926;&#127932;&#127933;&#127925;&#127926;&#127932;&#127933;&#127925;&#127926;&#127932;&#127933;&#127926;
+          </p>
+          <audio class="audio-container" controls autoplay loop>
+            <source src="./media/bg-music.mp3" type="audio/mpeg">
+            Your browser does not support the audio element.
+          </audio>
+          <p>
+            Enjoy the music while you're here~ music never fails to make the day better!
+          </p>
+          <p>
+            &#127925;&#127926;&#127932;&#127933;&#127925;&#127926;&#127932;&#127933;&#127925;&#127926;&#127932;&#127933;&#127926;
+          </p>
+        </td>
+      </tr>
+      <!-- form & image section -->
+      <tr>
+        <!-- form -->
+        <td>
+          <p> Fill in the form to create your account.</p>
+          <form id="registration-form" action="#registration-form" method="get">
+            &nbsp;&nbsp;
+            <label for="name">User Name:</label>
+            <input type="text" id="username" name="username" pattern="[a-z0-9_]{1,10}"
+              title="User name must be 1-10 characters long and can only contain lowercase letters, numbers, and underscores"
+              required placeholder="Enter your username">
+            <br><br>
```

Comment:
Sama ini juga, it would be better kaau di replace pake css

Reply:
oke noted mas. nanti tag <br>-nya aku ganti pake padding